### ISSUE TITLE
--verbose and "-- -" conflict

### ISSUE THREAD

**Issue by BenWiederhake** [meitar/git-archive-all.sh#29]
If using `--verbose`, then all messages are printed to stdout. This conflicts with "write output to stdout" (e.g. `git-archive-all.sh -- -`).

So the output of `-v` should go to stderr in this case. To keep things independent from each other, I suggest that the output of `-v` should *always* go to stderr, even when writing the archive/s to a file/dir.

**Comment by meitar** [comment]
Yeah, I think sending progress output to `stderr` is the right thing to do, too. Should be an easy patch, yes?
